### DISCUSSION TITLE
Welcome to mars Discussions!

### DISCUSSION THREAD

**Discussion by schoebel** [schoebel/mars#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marvineq** [comment]
Dear Mr. Schoebel. I use MARS on 3 pairs of servers for 2 years. And it works perfectly. Thanks a lot! 
My infrastructure looks like this: 

Server A iscsi target 
    - half of a storage capacity is primary LUN ( LVM standard logical volume) replicated to LVM thin LV on server B 
    - scheduled snapshots of replicated LUN are created on server B  at thin LV 
    - 2x SSD for MARS cache
     
Server B iscsi target 
    - vice versa  

This creates two active storages ready to provide failover.  A few scripts to enable easy export of snapshots via iscsi to linux or vmware and keep correct snapshot retention. If anyone is interested, I can share more details.
Regards, Vaclav

**Comment by schoebel** [comment]
hi Vaclav,

if you are using thin storage, you might benefit from the future DISCARD operation which not yet implemented, but it is on my roadmap.

Currently my prio is WIP-prosumer but after rollout at 1&1 I will turn to DISCARD (hopefully).

Thanks for letting me know this.

Cheers,

Thomas
